webpack/move_to_foreman/components/common/table/components/Table.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table as PfTable } from 'patternfly-react';
+import EmptyState from '../../emptyState';
+import PaginationRow from '../../../../../components/PaginationRow/index';
+
+import TableBody from './TableBody';
+
+const Table = ({
+  columns,
+  rows,
+  emptyState,
+  bodyMessage,
+  children,
+  itemCount,
+  pagination,
+  onPaginationChange,
+  ...props
+}) => {
+  if (rows.length === 0 && bodyMessage === undefined) {
```

Comment:
could be a case that no data exists (rows), but the body message is exist or null?

Reply:
Yes! In this case it should show EmptyState.

Reply:
The implementation for it is inside the `TableBody` :-1: